eZ/Bundle/EzPublishCoreBundle/Command/ResizeOriginalImagesCommand.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Search\SearchHit;
+use eZ\Publish\Core\FieldType\Image\Value;
+use eZ\Publish\Core\IO\IOServiceInterface;
+use Liip\ImagineBundle\Binary\BinaryInterface;
+use Liip\ImagineBundle\Exception\Imagine\Filter\NonExistingFilterException;
+use Liip\ImagineBundle\Model\Binary;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ResizeOriginalImagesCommand extends ContainerAwareCommand
+{
+    const DEFAULT_ITERATION_COUNT = 25;
+
+    protected function configure()
+    {
+        $this->setName('ezplatform:resize-original-images')->setDefinition(
```

Comment:
Hmm, there is a chance we'll create more commands related to image handling (like clearing image variation cache), so I'd rather call it `ezplatform:images:resize-original`. We also thought about extracting Image-related stuff (including Placeholder generators) to a separate bundle, so separating naming like this makes even more sense for the future.

Of course maybe my choice of naming is not the best, so suggestions welcomed (ping @adamwojs)

Reply:
`ezplatform:images:resize-original` is fine for me